build.gradle

```diff
@@ -84,6 +88,24 @@ gradlePlugin {
 
 apply from: 'versionupdates.gradle'
 
+tasks.register('PatchExamples') {
+    doLast {
+        String regex = "(id\\s*?[\\\"|\\']edu\\.wpi\\.first\\.GradleRIO[\\\"|\\'].*?version\\s*?[\\\"|\\'])(.+?)([\\\"|\\'])";
+
+        def examplesFolder = file("$rootDir/examples")
+        examplesFolder.eachFile { File file ->
+            if (file.isDirectory() && file.name != '_archived') {
+                def buildGradleFile = new File(file, 'build.gradle')
+                if (buildGradleFile.exists() && buildGradleFile.isFile()) {
+                    def text = buildGradleFile.text
+                    text = text.replaceAll(regex, "id \"edu.wpi.first.GradleRIO\" version \"${version}\" ")
+                    buildGradleFile.text = text
+                }
+            }
+        }
+    }
+}

```

Comment:
I want this to always run.

Reply:
And the input is not a file, but a variable, and I've had some failure  with gradles property inputs, so it would be impossible to make this work that way as well.